webview/src/plots/components/App.test.tsx

```diff
@@ -14,6 +14,7 @@ import {
 import '@testing-library/jest-dom/extend-expect'
```

Comment:
While I don't see this in development, I get this error when I run `App.test.tsx`:

```
dvc-vscode-webview:test:     console.error
dvc-vscode-webview:test:       ERROR TypeError: Cannot read properties of undefined (reading '0')
dvc-vscode-webview:test:           at toPathString (/Users/juliegalvan/Desktop/workspace/vscode-dvc/node_modules/vega-voronoi/build/vega-voronoi.js:1672:18)
dvc-vscode-webview:test:           at Voronoi.transform (/Users/juliegalvan/Desktop/workspace/vscode-dvc/node_modules/vega-voronoi/build/vega-voronoi.js:1659:35)
dvc-vscode-webview:test:           at Voronoi.evaluate (/Users/juliegalvan/Desktop/workspace/vscode-dvc/node_modules/vega-dataflow/build/vega-dataflow.js:2287:24)
dvc-vscode-webview:test:           at Voronoi.run (/Users/juliegalvan/Desktop/workspace/vscode-dvc/node_modules/vega-dataflow/build/vega-dataflow.js:2264:19)
dvc-vscode-webview:test:           at View.evaluate (/Users/juliegalvan/Desktop/workspace/vscode-dvc/node_modules/vega-dataflow/build/vega-dataflow.js:1706:19)
dvc-vscode-webview:test:           at View.evaluate (/Users/juliegalvan/Desktop/workspace/vscode-dvc/node_modules/vega-view/build/vega-view.js:1299:54)
dvc-vscode-webview:test:           at View.run (/Users/juliegalvan/Desktop/workspace/vscode-dvc/node_modules/vega-dataflow/build/vega-dataflow.js:1820:50)
dvc-vscode-webview:test:           at /Users/juliegalvan/Desktop/workspace/vscode-dvc/node_modules/react-vega/lib/Vega.js:59:23
dvc-vscode-webview:test:           at /Users/juliegalvan/Desktop/workspace/vscode-dvc/node_modules/react-vega/lib/VegaEmbed.js:52:13
dvc-vscode-webview:test: 
dvc-vscode-webview:test:       at log$1 (../node_modules/vega-util/build/vega-util.js:121:21)
dvc-vscode-webview:test:       at View.error (../node_modules/vega-util/build/vega-util.js:142:31)
dvc-vscode-webview:test:       at View.error (../node_modules/vega-dataflow/build/vega-dataflow.js:2095:32)
dvc-vscode-webview:test:       at View.evaluate (../node_modules/vega-dataflow/build/vega-dataflow.js:1738:10)
dvc-vscode-webview:test:       at View.evaluate (../node_modules/vega-view/build/vega-view.js:1299:54)
dvc-vscode-webview:test:       at View.run (../node_modules/vega-dataflow/build/vega-dataflow.js:1820:50)
dvc-vscode-webview:test:       at ../node_modules/react-vega/lib/Vega.js:59:23
dvc-vscode-webview:test:       at ../node_modules/react-vega/lib/VegaEmbed.js:52:13
```

Still unsure of what's causing it 🤔

Reply:
something is attempting to access the first entry in an empty array?

Reply:
> That's related to the dimensions of plots. We have something like this for checkpoint plots:
That did the trick! Thanks for the help!